File: 2차 과제/3-6/py 소스파일/3-6.py
from struct3_6 import polynomial
from func_list3_6 import addpoly
from func_list3_6 import print_func

def main() :
    a = polynomial(3, [4,3,5,0])
    b = polynomial(4, [3,1,0,2,1])
    c = polynomial(0, [0]*(max(a.Degree, b.Degree)+1))

    addpoly(a,b,c)
    print("A(x)= ", end="")
    print_func(a)

    print("A(x)= ", end="")
    print_func(b)
    
    print("A(x)= ", end="")
    print_func(c)

if __name__ == "__main__":
    main()

File: 2차 과제/3-6/c 소스파일/3-6.c
#include <stdio.h>
#include "3-6.h"

int main() {
    polynomial a={3, {4, 3, 5, 0}};
    polynomial b={4, {3, 1, 0, 2, 1}};
    polynomial c;
    
    addpoly(a, b, &c);
    printf("A(x)= ");
    print(a);

    printf("B(x)= ");
    print(b);

    printf("C(x)= ");
    print(c);

    return 0;
}